database/migrations/2017_03_20_205359_create_new_events_table_schema.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+
+class CreateNewEventsTableSchema extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('events', function (Blueprint $table) {
+            $table->increments('id');
+            $table->integer('eventable_id');
+            $table->string('eventable_type');
+            $table->string('content');
```

Comment:
This might make sense as a [JSON field](https://laravel.com/docs/5.3/migrations#columns), which I think would let Laravel automatically cast arrays into this column, and also let us (not sure on performance here) make `WHERE` queries based on the JSON content.